### ISSUE TITLE
Invalid numeric literal

### ISSUE THREAD

**Issue by danielcembranelli** [andersondanilo/CnabPHP#98]
Boa tarde pessoal, 

Ao atualizar meu arquivos começou a apresentar o seguinte erro:

FatalThrowableError in Detalhe.php line 358:
Parse error: Invalid numeric literal

Alguem ja passou por algo parecido?

**Comment by hiphoplucas** [comment]
Sim, na verdade você deve remover o 0 (zero) a esquerda dos 'ifs' do número 01 até 09. Isso passou a dar erro por conta de uma atualização da versão do PHP.

Olha uma explicação melhor: https://stackoverflow.com/questions/40735963/parse-error-invalid-numeric-literal
